.circleci/getChangelogs.sh

```diff
@@ -32,23 +32,46 @@ function cloneRepository() {
     old_version=$2
     new_version=$3
     repository_url=$4
-    echo "[OK] $project_name($repository_url) : $old_version -> $new_version"
+    echo "[OK] checkout: $project_name($repository_url)"
```

Comment:
Should this line come only after the `git clone`, if it is successful?

Reply:
The purpose was to say that it found the dependency on the requirements file and is now checking out this repo.